smoketests/pages/index.py

```diff
@@ -0,0 +1,24 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.base import Page
+from pages.language_picker import LanguagePickerPage
+
+
+class IndexPage(Page):
+    _page_title = 'Mozilla Support'
+    _url = '/'
+
+    _locale_picker = (By.CSS_SELECTOR, '.locale-picker')
+
+    @property
+    def current_language(self):
+        return self.selenium.find_element(*self._locale_picker).text
```

Comment:
Yeah, you could do that. I mostly threw enough together to move that language picker test over. I didn't want to spend much time making architecture decisions for a future I didn't think I knew much about.